### PR TITLE
css edits draft one

### PR DESCRIPTION
**Pull request by IbrahimPala1**
basic first draft

### CODE REVIEW

File: views/layout.hbs [hunk 1]
```diff
@@ -5,18 +5,30 @@
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
+  
     <div class="navigation-menu">
+      <div>
```
Comment (michaelcychan) [inline views/layout.hbs:10]: nice use of image to replace a button!

File: views/posts/index.hbs [hunk 2]
```diff
@@ -4,17 +4,19 @@
 
   <div class="post-container">
     <li class="each-message">{{this.message}}<br>{{this.date}}</li>
+    <br>
       <form action="/posts/delete" method="post" class="delete" >
         <input name="_id" type="hidden" value="{{this._id}}">
-        <input name="delete" type="submit" value ="🗑️">
+        <input style="background-color=transparent" name="delete" type="submit" value ="🗑️">
```
Comment (michaelcychan) [inline views/posts/index.hbs:10]: nice!

File: views/layout.hbs [hunk 3]
```diff
@@ -5,18 +5,30 @@
...
+        <a href="/">
+         <img alt="Logo for Acebook" src='/images/acebook_logo_2.png'
+         width="150" height="145">
+        </a>
+      </div>
+      <br>
+      <br>
       {{#if session.user.email }}
-      📧 E-mail: {{session.user.email}}
+      <h4>📧 Account:</h4>{{session.user.email}}
```
Comment (michaelcychan) [inline views/layout.hbs:19]: nice use of heading tag

**Review by michaelcychan — APPROVED** [review]
Looks beautiful!

File: views/layout.hbs [hunk 4]
```diff
@@ -5,11 +5,22 @@
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
+  
     <div class="navigation-menu">
+      <div> 
+        <a href="/">
+         <img alt="Logo for Acebook" src='/images/acebook_logo_2.png'
+         width="150" height="145">
+        </a>
```
Comment (kaykakaraka) [inline views/layout.hbs:14]: love the logo

File: public/stylesheets/style.css [hunk 5]
```diff
@@ -53,7 +62,7 @@ div.navigation-menu {
 
 .navigation-button:hover {
   color: greenyellow;
-  background-color: darkgreen;
+  background-color: rgb(21, 119, 206);
   padding-top: 3px;
 }
```
Comment (kaykakaraka) [inline public/stylesheets/style.css:68]: looking good

**Review by kaykakaraka — APPROVED** [review]
Nice design!

**Pull request merged**